### PR TITLE
Add custom webcomponent renovate UI

### PR DESCRIPTION
**Pull request by ger-benjamin**
Use latest ngeo and add feeback and swisscom-heatmap plugin.
Not working, dont' merge !

### CODE REVIEW

File: ui/vite.config.js [hunk 1]
```diff
@@ -14,8 +15,9 @@ let build_config =
 export default defineConfig({
   server: {
     port: 3002,
-    //https: true,
+    https: true,
```
Comment (sbrunner) [inline ui/vite.config.js:18]: The readme should probably be updated :-)

File: geoportal/vars.yaml [hunk 2]
```diff
@@ -918,6 +920,7 @@ vars:
         - http://localhost:3000
         - http://localhost:3001
         - http://localhost:3002
```
Comment (sbrunner) [inline geoportal/vars.yaml]: From me, we can remove this line :-)

Comment (ger-benjamin) [reply geoportal/vars.yaml]: Currently, this PR is a draft and don't work, but yes, if we want to use https, we can then remove this line

File: ui/vite.config.js (hunk 1 above)
Comment (ger-benjamin) [reply ui/vite.config.js:18]: done

File: geoportal/vars.yaml (hunk 2 above)
Comment (ger-benjamin) [reply geoportal/vars.yaml]: done

**Review by sbrunner — APPROVED** [review]

File: ui/package.json [hunk 3]
```diff
@@ -15,12 +15,13 @@
...
   "devDependencies": {
     "@rollup/plugin-inject": "5.0.5",
+    "@vitejs/plugin-basic-ssl": "^1.1.0",
     "sass-embedded": "1.79.4",
     "typescript": "5.5.3",
     "vite": "5.4.8"
   },
   "dependencies": {
-    "ngeo": "2.9.0-latest.20241030T072255Z.1c68af25a"
+    "ngeo": "2.9.0-latest.20241029T155214Z.b0ee1737f"
```
Comment (sbrunner) [inline ui/package.json]: This is a downgrade!

Comment (ger-benjamin) [reply ui/package.json]: was not one this morning.
Thanks

Comment (sbrunner) [reply ui/package.json]: Effectively: https://github.com/camptocamp/demo_geomapfish/commit/f1c25b25d18555d5847b8293e8c33342cf844193 :-)

**Pull request merged**